plotsquared/SUMMARY.md

```diff
@@ -0,0 +1,63 @@
+# Summary​
+
+* [Why should you switch to PlotSquared?](./why-plotsquared.md)
+
+## Installation
+
+* [Installation](./installation/installation.md)
+* [Migrating from an older release](./installation/migrating-from-an-older-major-release.md)
+* [Updating from 1.12 to 1.13](./installation/updating-from-1.12-to-1.13.md)
```

Comment:
I think this can be skipped. The documentation is for unmaintained, ~5-year-old versions.